src/vs/workbench/contrib/debug/common/debugModel.ts

```diff
@@ -1244,6 +1244,30 @@ export class DebugModel implements IDebugModel {
 		}
 	}
 
+	/**
+	 * Update the call stack and notify the call stack view that changes have occurred.
+	 */
+	async fetchAndRefreshCallStack(thread: Thread, levels?: number): Promise<void> {
```

Comment:
Honestly I think this one should be called `fetchCallstack` and the other could be called `refreshTopOfCallstack` or something like that. I also think that ideally callstackview would call this, but that could be a refactoring that I do later.

Reply:
why should callstackview call it? is there info that it needs from there?

Reply:
But it seems that the debug model is already managing fetching the callstack here? What does this have to do so with the callstackview calling this function?

Reply:
ohh, you mean calling this instead of `Thread`'s `fetchCallStack`?